canonical-web-and-design/canonical.com
# 2568 find a partner

## Done

Updated the copy and left-aligned the filter button text

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8002/partners/find-a-partner
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the intro copy matches the [copydoc](https://docs.google.com/document/d/1R_iDdmYjI3DOK2LnlkorJXWaf_OrOOJmpZGAPOlLTBM/edit)
- Check that the filter button looks good at all viewports

## Issue / Card

Fixes #2568
